doc/man1/flux-modprobe.rst

```diff
@@ -195,9 +195,23 @@ MODULE CONFIG
 =============
 
 :program:`flux modprobe` loads information about Flux modules from the
-``modules`` array in ``modprobe.toml``. Module configuration may be
-extended by dropping new TOML files into ``modprobe.d`` in the modprobe
-configuration directory, typically ``/etc/flux/modprobe``.
+``modules`` array in ``modprobe.toml``.
+
+Module configuration may be extended by dropping new TOML files into a
+``modprobe.d`` directory in the modprobe search path. The default search
+path is::
+
+  $datadir/flux/modprobe:$sysconfdir/flux/modprobe
+
+which is typically::
+
+  /usr/share/flux/modprobe:/etc/flux/modprobe
+
+Packages should install files under ``/usr/libexec/flux/modprobe`` while
+site updates and modifications go under ``/etc/flux/modprobe``.
+
+The default search path can be overridden using :envvar:`FLUX_MODPROBE_PATH`
+or extended using :envvar:`FLUX_MODPROBE_PATH_APPEND`
```

Comment:
FLUX_MODPROBE_PATH_APPEND should be added to flux-environment(7) too!

Reply:
Thanks for catching that!